survey.tex

```diff
@@ -2,6 +2,11 @@
 % Responsible : Zeljko
 \section{Survey Cadence}
 
-Early Science observations should align as closely as possible with main survey and ultimate long-term science goals
-Details on the implications for the Survey Cadence of the \esp will be added in future as the outcome of commissioning becomes clear.  
+% Early Science observations should align as closely as possible with main survey and ultimate long-term science goals
+% Details on the implications for the Survey Cadence of the \esp will be added in future as the outcome of commissioning becomes clear.
 
+The early science observations can be separated into two kinds: commissioning observations, and survey observations taken during the first 6 months of the LSST.
+Only the latter---the ``early survey observations`` will be included in the LSST data processing.
+Optimizing the early survey observing schedule for incremental template generation and alert production in the first 6 months will mean that the time sampling in about half the LSST survey region in Year 1 may look somewhat different to that in subsequent years.
```

Comment:
I am surprised that this is a statement only after the first 6 months. I would have thought the statement would apply to the first 12 months since there likely would not be complete sky coverage after the first 6 months due to Earth blocking a substantial fraction of the sky.

Reply:
Again - @drphilmarshall , what is  your thinking here?

Reply:
The first 6 months of LSST is what goes into DR1, and the Early Science period, by our definition, only runs through DR1. And I was trying to focus on the Early Science data.  We can just as well write a sentence about optimizing the Y1 observing schedule, no problem. I'll try something.

Reply:
Fixed this in the same commit as above. Resolving this, will leave the screenshot of the updated text.